docs/tutorials/integrations/ghactions.md

```diff
@@ -4,19 +4,19 @@ template: main.html
 
 # GitHub Actions
 
-There are two ways that you can use Iter8 with GitHub Actions. You can [run Iter8 CLI within a GitHub Actions workflow](#use-iter8-in-a-github-actions-workflow) and you can also [use Iter8 to trigger a GitHub Actions workflow](#use-iter8-to-trigger-a-github-actions-workflow) from an experiment.
+There are two ways that you can use Iter8 with GitHub Actions. You can [run Iter8 CLI within a GitHub Actions workflow](#use-iter8-in-a-github-actions-workflow) and you can also [use Iter8 to trigger a GitHub Actions workflow](#use-iter8-to-trigger-a-github-actions-workflow) during a performance test.
 
-# Trigger a GitHub Actions workflow from an Iter8 experiment
+# Trigger a GitHub Actions workflow from a performance test
 
 Iter8 provides a [`github` task](../../user-guide/tasks/github.md) that sends a [repository_dispatch](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#repository_dispatch) which can trigger the workflows in the default branch of a GitHub repository.
 
 ## Example
 
-In this example, you will run the [Your first performance test](../../getting-started/first-performance.md) but at the end of the experiment, Iter8 will trigger a workflow on GitHub.
+In this example, you will run the [Your first performance test](../../getting-started/first-performance.md) but at the end of the performance test, Iter8 will trigger a workflow on GitHub.
 
-In this simple example, the workflow will simply print out an experiment summary that it will receive with the `repository_dispatch`. In a more sophisticated scenario, the workflow could, for example, read from an experiment summary and determine what to do next.
+In this simple example, the workflow will simply print out a test summary that it will receive with the `repository_dispatch`. In a more sophisticated scenario, the workflow could, for example, read from test test summary and determine what to do next.
```

Comment:
```suggestion
In this simple example, the workflow will simply print out a test summary that it will receive with the `repository_dispatch`. In a more sophisticated scenario, the workflow could, for example, read from the test summary and determine what to do next.
```